src/QMCWaveFunctions/RotatedSPOs.h

```diff
@@ -94,19 +104,23 @@ class RotatedSPOs : public SPOSet, public OptimizableObject
 
   // This function applies the global rotation (similar to apply_rotation, but for the full
   // set of rotation parameters)
-  void applyFullRotation(const std::vector<RealType>& full_param, bool use_stored_copy);
+  void applyFullRotation(const std::vector<ValueType>& full_param, bool use_stored_copy);
 
-  // Compute matrix exponential of an antisymmetric matrix (result is rotation matrix)
+  // Compute matrix exponential of an antisymmetric/antihermitian matrix for
+  // the real/complex case.  Overwrites matrix with its exponential (result is rotation matrix)
   static void exponentiate_antisym_matrix(ValueMatrix& mat);
 
-  // Compute matrix log of rotation matrix to produce antisymmetric matrix
-  static void log_antisym_matrix(const ValueMatrix& mat, ValueMatrix& output);
+  // Compute matrix log of rotation matrix to produce antisymmetric matrix.  Technically, this
+  // performs the log of any matrix where the eigenproblem is stable, but we only take logs of
+  // matrices of the form A=e^K, where K is antisymmetric/antihermitian.
+  static void log_antisym_matrix(const Matrix<RealType>& mat, Matrix<RealType>& output);
+  static void log_antisym_matrix(const Matrix<std::complex<RealType>>& mat, Matrix<std::complex<RealType>>& output);
 
   //A particular SPOSet used for Orbitals
   std::unique_ptr<SPOSet> Phi;
 
   /// Set the rotation parameters (usually from input file)
-  void setRotationParameters(const std::vector<RealType>& param_list);
+  void setRotationParameters(const std::vector<ValueType>& param_list);
```

Comment:
Just leaving a note, based on our previous discussion this will likely need to end up being changed back to RealType since it interfaces with SPOSetBuilder, and SPOSetBuilder explicitly passes this in as RealType. However, this can be fixed when we fully enable complex

Reply:
Agreed.  For now, this is coded to conform to current complex handling of optimizable types.  When we decide to do something about it, I will fix this and the parameter dependent unit tests.